File: app/src/main/java/com/example/yooz/myapplication/FragmentPagerActivity.java
package com.example.yooz.myapplication;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class FragmentPagerActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_fragment_pager);

        if(savedInstanceState == null){
            FragmentManager fm = getSupportFragmentManager();
            FragmentTransaction ft = fm.beginTransaction();
            Fragment fragment = MyFragment.newInstance(1);
            ft.add(R.id.container, fragment);
            ft.commit();

        }
    }
}

File: app/src/main/java/com/example/yooz/myapplication/FragmentPagerActivity.kt
package com.example.yooz.myapplication

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class FragmentPagerActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fragment_pager)

        if (savedInstanceState == null) {
            val fm = supportFragmentManager
            val ft = fm.beginTransaction()
            val fragment = MyFragment.newInstance(1)
            ft.add(R.id.container, fragment)
            ft.commit()

        }
    }
}
